src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -7,6 +7,46 @@ DROP TABLE IF EXISTS clinical_event_derived;
 DROP TABLE IF EXISTS genetic_alteration_derived;
 DROP TABLE IF EXISTS generic_assay_data_derived;
 
+-- the following query "fixes" the sample_profile table by adding entrees for "missing" samples -- those which appear in mutated case list but not in the MySQL sample_profile table
```

Comment:
"entrees" should probably be replaced with "entries", but see later comment about simplifying comments explaining the problem being fixed. I suggested a link to external documentation / explanations for this complex topic rather than putting explanations in the code.

Reply:
Listen @sheridancbio, I meant what i said and I want a Filet Mignon or Salmon for every missing sample, in perpetuity.